test/eventlistener_test.go

```diff
@@ -46,8 +47,19 @@ const (
 	resourceLabel = triggersv1.GroupName + triggersv1.EventListenerLabelKey
 	triggerLabel  = triggersv1.GroupName + triggersv1.TriggerLabelKey
 	eventIDLabel  = triggersv1.GroupName + triggersv1.EventIDLabelKey
+
+	examplePRJsonFilename = "pr.json"
 )
 
+func loadExamplePREventBytes() ([]byte, error) {
```

Comment:
This is great! I think we should have more tests that use actual event payload...and not just in the e2e test but also in the sink tests!